### PR TITLE
Make blender input names clearer

### PR DESCRIPTION
**Pull request by Zelllll**
Thanks to @sauraen for helping with these

### CODE REVIEW

File: fast64_internal/f3d/f3d_enums.py [hunk 1]
```diff
@@ -254,22 +254,22 @@
...
 
 enumBlendColor = [
-	('G_BL_CLR_IN', 'Input', 'Input'),
-	('G_BL_CLR_MEM', 'Memory', 'Memory'),
-	('G_BL_CLR_BL', 'Blend', 'Blend'),
-	('G_BL_CLR_FOG', 'Fog', 'Fog')
+	('G_BL_CLR_IN', 'Input (CC/Blender)', 'First cycle: Color Combiner RGB, Second cycle: Blender numerator from first cycle'),
+	('G_BL_CLR_MEM', 'Framebuffer', 'Framebuffer (Memory)'),
+	('G_BL_CLR_BL', 'Blend', 'Blend Register'),
+	('G_BL_CLR_FOG', 'Fog', 'Fog Register')
```
Comment (Dragorn421) [inline fast64_internal/f3d/f3d_enums.py:260]:
```suggestion
	('G_BL_CLR_BL', 'Blend Color', 'Blend Color Register'),
	('G_BL_CLR_FOG', 'Fog Color', 'Fog Color Register')
```
the manual likes to call thse registers "color registers" (eg in http://n64devkit.square7.ch/n64man/gdp/gDPSetFogColor.htm )
and I think it's clearer anyway to mention "Color"

File: fast64_internal/f3d/f3d_enums.py [hunk 2]
```diff
@@ -254,22 +254,22 @@
...
 ]
 
 enumBlendAlpha = [
-	('G_BL_A_IN', 'Input', 'Input'),
-	('G_BL_A_FOG', 'Fog', 'Fog'),
-	('G_BL_A_SHADE', 'Shade', 'Shade'),
+	('G_BL_A_IN', 'Color Combiner Alpha', 'Color Combiner Alpha'),
+	('G_BL_A_FOG', 'Fog Alpha', 'Fog Register Alpha'),
```
Comment (Dragorn421) [inline fast64_internal/f3d/f3d_enums.py:265]:
```suggestion
	('G_BL_A_FOG', 'Fog Alpha', 'Fog Color Register Alpha'),
```

**Review by Dragorn421 — APPROVED** [review]

**Comment by Zelllll** [comment]
Would it be a good idea to change the combiner to also say "Texture 0 Color" etc in this PR? I didn't include the word "Color" before to be consistent with the combiner UI, but I think mentioning color there as well would make it clearer.

**Comment by Zelllll** [comment]
Alright I've made the changes to the combiner UI, as well as fixed another blender name error

**Review by Dragorn421 — APPROVED** [review]

**Pull request merged**